pkgs/tools/package-management/disnix/dysnomia/default.nix

```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, fetchurl, netcat
-, systemd ? null, ejabberd ? null, mysql ? null, postgresql ? null, subversion ? null, mongodb ? null, mongodb-tools ? null, influxdb ? null, supervisor ? null, docker ? null
+, systemd ? null, ejabberd ? null, mysql ? null, postgresql ? null, subversion ? null, mongodb ? null, mongodb-tools ? null, influxdb ? null, supervisor ? null, docker ? null, nginx ? null, s6-rc ? null, xinetd ? null
```

Comment:
It wouldn't hurt to add a line break here.

Reply:
yes maybe. I'll see if I can change that